src/TableCell/tableCell.css

```diff
@@ -2,8 +2,29 @@
 
 $shark: 768px;
 
+@media( min-width: $shark )
+{
+	.default
+    {
+    	padding-bottom:	0;
+
+    	&:last-child
+    	{
+        	padding-bottom:	-1rem;
+    	}
+
+        &:first-child
+        {
+            padding-left: 0;
+        }
+    }
+}
```

Comment:
I see that the above code block came from `tableRow.css`. But it makes no sense here. (Also, negative `padding` isn‘t valid CSS 😛 )

This stuff should be removed, except for the `&:first-child` bit which should move to the `.default` block below.